proxy-init/cmd/root.go

```diff
@@ -34,10 +34,10 @@ func NewRootCmd() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "proxy-init",
-		Short: "Adds a Kubernetes pod to the Conduit Service Mesh",
-		Long: `proxy-init Adds a Kubernetes pod to the Conduit Service Mesh.
+		Short: "proxy-init adds a Kubernetes pod to the Linkerd service mesh",
+		Long: `proxy-init adds a Kubernetes pod to the Linkerd service mesh.
 
-Find more information at https://conduit.io/.`,
+Find more information at https://linkerd.io/.`,
```

Comment:
Unfortunately. this isn't yet the case. We should continue to guide people to conduit.io/docs for now... But I might even just delete this altogether.  You have to pretty far in the deps of this project to encounter this.